docs/content/querying/dimensionspecs.md

```diff
@@ -308,6 +308,37 @@ This allows distinguishing between a null dimension and a lookup resulting in a
 For example, specifying `{"":"bar","bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["bar", "oof", "baz"]`.
 Omitting the empty string key will cause the missing value to take over. For example, specifying `{"bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["oof", "oof", "baz"]`.
 
+### Cascade Extraction Function
+
+Provides chained execution of extraction functions.
+
+A property of `extractionFns` contains an array of any extraction functions, which is executed in the array index order.
+
+Example for chaining [regular expression extraction function](#regular-expression-extraction-function), [javascript extraction function](#javascript-extraction-function), and [substring extraction function](#substring-extraction-function).
+
+```json
+{
+  "type" : "cascade", 
+  "extractionFns": [
+    { 
+      "type" : "regex", 
+      "expr" : "/([^/]+)/", 
+      "replaceMissingValues": false, 
+      "replaceMissingValuesWith": null
+    },
+    { 
+      "type" : "javascript", 
+      "function" : "function(str) { return \"the \".concat(str) }" 
+    },
+    { 
+      "type" : "substring", 
+      "index" : 0, "length" : 7 
+    }
+  ]
+}
+```
+
```

Comment:
it would be really cool if the example showed the dimension value and the final transformed value